docs/apis-tools/administration-api/authentication.md

```diff
@@ -59,6 +59,18 @@ curl --header "Content-Type: application/json" \
 Access tokens have a validity period found in the access token. After this time, a new access token must be requested.
 :::
 
-:::note
-The auth service has built-in rate limiting. If too many token requests are executed in a short time, the client is blocked for a certain time. Since the access tokens have a certain validity period, they must be cached on the client side.
-:::
+## Built-in rate limit
+
+The OAuth service has a built-in rate limit of about one request per second for all clients with the same source IP address.
```

Comment:
@christinaausley it's a minor change, but I think I would suggest to completely remove the wording `built-in`: I'm not sure why it was there in the first place and it doesn't indicate anything really useful:

```suggestion
## Rate limit

The OAuth service rates limit all clients with the same source IP address to about one request per second.
```

Reply:
Done 👍